src/sentry/lang/javascript/processor.py

```diff
@@ -220,9 +220,12 @@ def fetch_file(url, project=None, release=None, allow_scraping=True):
 
     Attempts to fetch from the cache.
     """
-    cache_key = 'source:cache:v2:%s' % (
-        md5(url).hexdigest(),
-    )
+    if url is None:
```

Comment:
IMO passing None to "fetch this file" is just bad code elsewhere. If you think about it in a staticly typed language this would be a compile error. IMO it should be the same kind of thing for us.